Changelog.md

```diff
@@ -6,6 +6,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+### Added
+- [Breaking change] ID Validation, ensure the ID coming back has been incremented using the configured `auto_increment_increment`. (https://github.com/zendesk/global_uid/pull/63)
```

Comment:
This is a breaking change as any client upgrading that doesn't have their `GlobalUid::Base.global_uid_options[:increment_by]` set to be the same value of the `auto_increment_increment` within the database will start to see exceptions (unless suppressed). This value was previously only used in [this check](https://github.com/zendesk/global_uid/pull/63/files#diff-a485bda32508290d377722bde6530880L77-R79)